xgbgen/main.go

```diff
@@ -49,16 +46,26 @@ func main() {
 	c := newContext()
 	c.Morph(xmlBytes)
 
+	outFile, err := os.Create(fmt.Sprintf("%s/%s.go", *packageName, *packageName))
+	if err != nil {
+		panic(err)
+	}
+	outFile.Truncate(0)
```

Comment:
`os.Create` already includes the flag `O_TRUNC` https://cs.opensource.google/go/go/+/refs/tags/go1.23.0:src/os/file.go;l=374

Reply:
Fixed by cbed66aa3ac97f6bb2f59ad18df2472a2d4368bd